src/modules/collection/sagas.spec.ts

```diff
@@ -37,16 +41,30 @@ import {
   publishCollectionSuccess,
   publishCollectionFailure,
   SAVE_COLLECTION_SUCCESS,
-  SAVE_COLLECTION_FAILURE
+  SAVE_COLLECTION_FAILURE,
+  initiateTPApprovalFlow
 } from './actions'
 import { collectionSaga } from './sagas'
 import { Collection } from './types'
 import { getLatestItemHash, UNSYNCED_COLLECTION_ERROR_PREFIX } from './utils'
-import { getOpenModals } from 'decentraland-dapps/dist/modules/modal/selectors'
+
+jest.mock('./utils', () => ({
+  ...jest.requireActual('./utils'),
+  isTPDeployEnabled: jest.fn().mockImplementation(() => true)
+}))
```

Comment:
question: are we trying to mock all functions from `./utils` here or is it used more as a way to mock isTPDeployEnabled.

I'm asking because we were already importing `import { getLatestItemHash, UNSYNCED_COLLECTION_ERROR_PREFIX } from './utils'` here but now those will be mocked and if we just need to mock `isTPDeployEnabled` we can do that on a top-level `beforeAll`

Reply:
Let's continue it in Slack :)